test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -152,7 +152,7 @@ func (s *snapshottableTestSuite) defineTests(driver TestDriver, pattern testpatt
 		framework.ExpectNoError(err)
 
 		// Get the bound PV
-		pv, err := cs.CoreV1().PersistentVolumes().Get(pvc.Spec.VolumeName, metav1.GetOptions{})
+		_, err = cs.CoreV1().PersistentVolumes().Get(pvc.Spec.VolumeName, metav1.GetOptions{})
```

Comment:
I'm undecided on whether or not we should remove the [Feature] tag from the tests. Because this also has a deployment dependency, the test is going to start failing for deployments that haven't been updated. I'll try to talk to a few folks to figure out what the recommendation should be.

Reply:
Sure.

Reply:
Didn't get much response. I would say let's keep the feature tag on, and we'll look into adding another non-alpha job for it.